index.js

```diff
@@ -100,16 +100,18 @@ try {
 
 // eslint-disable-next-line no-redeclare
 const CONFIG = {
```

Comment:
Could we potentially just put all of the settings in the same localstorage JSON rather than having a different one for each Setting option?

Reply:
Hmm, maybe. Then we'd have to parse the entire thing if we want to pull one out (for example, to see if stars are enabled in Card.js), but that's not super difficult. I'd suggest adding a new issue or PR for that if you like since it's not super related to this functionality.

Reply:
Will do